UI/src/main/webapp/index.html

```diff
@@ -0,0 +1,313 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Profit!</title>
+		<link rel="stylesheet" type="text/css" href="resources/css/styles.css">
+	</head>
+	<body>
+<!--		<script src="script.js"></script>-->
+		<script src="resources/js/AdList.js"></script>
+		<script>
+			const adList = new AdList([
+				{
+					id: '1',
+					description: '������ �� ������ - �� 15%',
+					createdAt: new Date('2021-03-15T23:00:00'),
+					link: 'https://coolchairs.com',
+					vendor: 'Chair Service',
+					photoLink: 'https://images.app.goo.gl/dgAFyP2cEduzkJUP9',
+					validUntil: new Date('2021-06-01T23:00:00'),
+					discount: '15%',
+					hashTags: ['tag1', 'tag2']
+				},
+
+				{
+					id: '2',
+					description: '��� ��� ����� ������  -  �������� ������� ������������ �������� ����� (������������) ������� � ������������ ������� �������� ������, ��������������� �������� ������������.',
```

Comment:
This is not looking pretty)) Please always re-check your commits before raising a pull-request